learn/configuration/settings.md

```diff
@@ -216,3 +217,22 @@ Suppose you have an e-commerce dataset. For an index that contains information a
 <CodeSamples id="settings_guide_synonyms_1" />
 
 By doing so, when searching for `black sweater`, results for `black jumper` will also be returned.
+
+## Typo tolerance
+
+The typo tolerance settings for an index. Typo tolerance allows users to make mistakes when typing and still find something. [Disabling typo tolerance will not impact the `typo` ranking rule.](/learn/configuration/typo_tolerance.md#impact-of-typo-tolerance-on-the-typo-ranking-rule)
+
+The `typoTolerance` object is composed of the following fields:
```

Comment:
I'm not sure if this is the best way to summarize this, but any suggestions are welcome

Reply:
I would have slightly changed the first sentence.

"The typo tolerance settings for an index. Typo tolerance allows users to make mistakes when typing and still find documents containing words close to their typed query"